src/Parser/DeathKnight/Unholy/Modules/Items/ColdHeartEfficiency.js

```diff
@@ -20,63 +20,117 @@ class ColdHeartEfficiency extends Analyzer {
 
   totalColdHeartCasts = 0;
   correctColdHeartCasts = 0;
-  buffColdHeart = 0;
+  buffColdHeart = 20; //Stacks start at 20 at start of fight
+  
   unholyStrengthStart = 0;
   unholyStrengthRemaining = 0;
+  
+  concordanceStart = 0;
+  concordanceRemaining = 0;
+  
+  khazgorothStart = 0;
+  khazgorothRemaining = 0;
+  
+  timeAtMaxStacksStart = this.owner.fight.start_time;
+  timeAtMaxStacks = 0;
+  
+  unholyStrengthDuration = 15000;
+  concordanceDuration = 10000;
+  khazgorothDuration = 15000;
+  
+  coldHeartMaxStack = 20;
+  coldHeartOptimalMinStack = 14;
+	
+  remainingDurationAllowed = 4000;
+  maxDurationAtMaxStacksAllowed = 4000;
```

Comment:
All 7 of the above constants should be static consts, not class variables. For example this one should be `const MAX_DURATION_AT_MAX_STACKS_ALLOWED = 4000`, defined like that above the class declaration.